pkg/stconfig/keygen.go

```diff
@@ -0,0 +1,19 @@
+package stconfig
```

Comment:
To make it a bit more compact, can you put these four functions in one file `pkg/stconfig/stconfig.go`?